src/content/developers/docs/consensus-mechanisms/pow/mining/index.md

```diff
@@ -66,7 +65,9 @@ Watch Austin walk you through mining and the proof-of-work blockchain.
 
 ## The mining algorithm {#mining-algorithm}
 
-The Ethereum mining algorithm has undergone several upgrades since its inception. The original algorithm, "Dagger Hashimoto" was based around the provision of a large, transient, randomly generated dataset which forms a [Directed Acyclic Graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph) (the Dagger-part), with miners attempting to solve a particular constraint on it, partly determined through a block’s header-hash. This algorithm was novel because it had high memory-access bandwidth requirements but could be run using a modest processor, making it GPU-friendly but resistant to the type of ASIC-driven hardware arms race that could pose a centralization risk (more on [problems with ASICS](https://www.investopedia.com/investing/why-centralized-crypto-mining-growing-problem/)). After substantial upgrades to the algorithm, it was renamed to "Ethash". This renaming happened before mining began on Ethereum mainnet. Dagger-Hashimoto was a precursor, research algorithm that was not used on Ethereum mainnet.
+Ethereum Mainnet only ever used one mining algorithm - ['Ethash'](/developers/docs/consensus-mechanisms/pow/mining/mining-algorithms/ethash). This was a successor to an original R&D algorithm known as ['Dagger-Hashamoto'](/developers/docs/consensus-mechanisms/pow/mining/mining-algorithms/dagger-hashamoto).
+
+The algorithm is based around the provision of a large, transient, randomly generated dataset which forms a [Directed Acyclic Graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph) (the Dagger part), with miners attempting to solve a particular constraint on it, partly determined through a block’s header-hash. This has high memory-access bandwidth requirements but could be run using a modest processor, making it GPU-friendly but resistant to the type of ASIC-driven hardware arms race that could pose a centralization risk (more on [problems with ASICS](https://www.investopedia.com/investing/why-centralized-crypto-mining-growing-problem/)). ASICs for Ethereum mining were eventually developed, but the majority of mining on Ethereum was nevertheless done using GPUs.
```

Comment:
Can we just remove this section @jmcook1186? 

The only nice thing I'd like to keep **somewhere** is the problem with ASICS information, though if it doesn't already, I think that can be covered in the mining algorithms page.

Reply:
Coming back to this, I feel the same way. I've removed it and [created a card to add it to the mining algorithms page.](https://www.notion.so/efdn/Add-information-about-the-problem-with-ASICS-to-mining-algorithms-page-45985053907a4a50a2049e941c9c5cba)